docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3.9'
+services:
+  dynamodb:
+    image: 'amazon/dynamodb-local:latest'
+    command: '-jar DynamoDBLocal.jar -sharedDb -dbPath ./data'
+    container_name: dynamodb-local
+    restart: always
+    ports:
+      - 8000:8000
+    volumes:
+      - './docker/dynamodb:/home/dynamodblocal/data'
+    working_dir: /home/dynamodblocal
+
+  app:
```

Comment:
Should this have environment variables for DYNAMO_REGION, DYNAMO_URL, DYNAMO_PORT?

Reply:
only the app service needs those except for DYNAMO_PORT